### ISSUE TITLE
Doesn't work on issue_comment event type

### ISSUE THREAD

**Issue by plieberg** [Tj-Stockopedia/action-get-changed-folders#135]
Would be really nice to have support for the issue_comment event type. This allows ad hoc runs by commenting on an open PR.

Current code only supports `push` and `pull_request`.

```
switch (context.eventName) {
      case 'push':
        innerContext = {
          repo: context.repo,
          after: context.payload.after,
          before: context.payload.before
        }
        break
      case 'pull_request':
        innerContext = {
          repo: context.repo,
          after: context.payload.pull_request!.head.sha,
          before: context.payload.pull_request!.base.sha
        }
        break
      default:
        throw new Error('Event type not supported')
    }
```
